Sdk.zig

```diff
@@ -90,14 +79,16 @@ pub fn init(b: *Builder, user_config: ?UserConfig, toolchains: ToolchainVersions
     const host_tools = blk: {
         const zip_add = b.addExecutable(.{
             .name = "zip_add",
-            .root_source_file = .{ .path = sdkRoot() ++ "/tools/zip_add.zig" },
+            .root_source_file = b.path("tools/zip_add.zig"),
```

Comment:
This change is breaking as it now only allows to compile this repository, but you can't use the `Sdk.zig` anymore from any other path except project root.

Until we gain ability to run it properly from a zig dependency, we have to ues `.{ .cwd_relative =  … }`

Reply:
That change is entirely in the last commit so it can be reverted. I would have added package manager support but I decided to use android studio instead of this